lib/manageiq/appliance_console/certificate.rb

```diff
@@ -40,8 +41,11 @@ def initialize(options = {})
     end
 
     def request
+      undo_tracking if complete?
```

Comment:
Be doing this first wouldn't we remove the self signed certs before saving them off? Or would we never get into the `if should_request_key?` block if we are `complete?` here?

Reply:
Thank you for the review input @carbonin.

This line: `undo_tracking if complete?` will only invoke `undo_tracking` if `getcert request`
was successful, which is what causes `complete?` to return true.
So any self signed certs would have already been saved off.